freeipa/src/main/java/com/sequenceiq/freeipa/controller/FreeIpaV1Controller.java

```diff
@@ -258,6 +259,13 @@ public OperationStatus repairInstances(@RequestObject @Valid RepairInstancesRequ
         return repairInstancesService.repairInstances(accountId, request);
     }
 
+    @Override
+    @CheckPermissionByRequestProperty(path = "environmentCrn", type = CRN, action = REPAIR_FREEIPA)
```

Comment:
is it intentional that the `rebuild` has an action from repair?

Reply:
Yes, rebuilding is a type of repair. But rebuilding the cluster requires more steps (e.g. delete freeipa, restore backup, etc.)